ci/publish-tarball.sh

```diff
@@ -45,7 +45,7 @@ linux)
   TARGET=x86_64-unknown-linux-gnu
   ;;
 windows)
-  TARGET=x86_64-pc-windows-gnu
+  TARGET=x86_64-pc-windows-msvc
```

Comment:
It used to be the case that rocksdb wouldn't build with the MSVC toolchain.  But this is clearly resolved as somehow the MSVC toolchain snuck in (Travis CI update perhaps?!) and we're definitely building with MSVC instead of GNU now